Aquality.Selenium.Core/src/Aquality.Selenium.Core/Utilities/ElementActionRetrier.cs

```diff
@@ -2,58 +2,36 @@
 using OpenQA.Selenium;
 using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Threading;
 
 namespace Aquality.Selenium.Core.Utilities
 {
     /// <summary>
     /// Retries an action or function when <see cref="HandledExceptions"/> occures.
     /// </summary>
-    public class ElementActionRetrier : IElementActionRetrier
+    public class ElementActionRetrier : ActionRetrier, IElementActionRetrier
     {
-        private readonly IRetryConfiguration retryConfiguration;
-
         /// <summary>
         /// Instantiates the class using retry configuration.
         /// </summary>
         /// <param name="retryConfiguration">Retry configuration.</param>
-        /// <param name="handledExceptions">Exceptions to be handled. 
-        /// If set to null, <see cref="StaleElementReferenceException"/> and <see cref="InvalidElementStateException"/> will be handled.</param>
-        public ElementActionRetrier(IRetryConfiguration retryConfiguration, IList<Type> handledExceptions = null)
+        public ElementActionRetrier(IRetryConfiguration retryConfiguration) : base(retryConfiguration)
```

Comment:
the parameter was useful, why have you removed it? I cannot agree with that
Now I'll need to define a custom implementation and override the virtual method - it was much simplier before

Reply:
The same as in java version

Reply:
Actually, we can add setter to HandledExceptions property

Reply:
yes, let's add it